test/aerospike_clj/mock_client_test.clj

```diff
@@ -1,7 +1,6 @@
 (ns aerospike-clj.mock-client-test
```

Comment:
For the whole file - please align `let` bindings to reduce the diff's size just because of whitespace changes.